app.js

```diff
@@ -1,3 +1,24 @@
+/* Object */
+const projectData = [{
+  title: 'Project Title',
+  lenguage1: 'HTML',
+  lenguage2: 'JavaScript',
+  lenguage3: 'CSS',
+  image: './Images/SnapshootPortfolio.png',
+  imageAlt: 'SnapshootPortfolio',
+  description: 'Lorem ipsum dolor, sit amet consectetur adipisicing elit. Consequatur amet quasi suscipit enim quidem quas odio libero totam nisi. Numquam corrupti sequi eligendi error laboriosam quasi nulla quidem eaque reiciendis.',
+},
+
+{
+  title: 'Project Title 2',
+  lenguage1: 'Ruby',
+  lenguage2: 'bootstrap',
+  lenguage3: 'Ruby on Rails',
+  image: './Images/SnapshootPortfolio.png',
+  imageAlt: 'SnapshootPortfolio',
+  description: 'Second Lorem ipsum dolor, sit amet consectetur adipisicing elit. Consequatur amet quasi suscipit enim quidem quas odio libero totam nisi. Numquam corrupti sequi eligendi error laboriosam quasi nulla quidem eaque reiciendis.',
+}];
+/* Functions */
```

Comment:
- [ ] Kindly ensure all your projects are in the array and are displayed on your page. Currently, you only have 1 project.
- [ ]  Kindly check the design of your popup, the Figma design has the background blurred and a bit dark kindly check on it here is the [link](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34%3A1278) to the design.